integrations/github.rst

```diff
@@ -0,0 +1,113 @@
+.. _integrations_github:
+
+|github_logo| GitHub
+====================
+
+.. include:: ../common/experimental_warning.inc
+
+The Conan `GitHub Actions <https://github.com/features/actions>`_ integration allows you to setup
+Conan client in your GitHub Actions workflows in a simple and effective way.
+
+The project can be found on its `GitHub marketplace page <https://github.com/marketplace/actions/setup-conan-environment>`_, or
```

Comment:
That link: https://github.com/marketplace/actions/setup-conan-environment is wrong... where did it came from?
```suggestion
The project can be found on its `GitHub marketplace page <https://github.com/marketplace/actions/setup-conan-client>`_, or
```